### ISSUE TITLE
improve security on audio download

### ISSUE THREAD

**Issue by fanick-ber** [NexdApp/nexd-backend#83]

**Comment by andre-stefanov** [comment]
we have to be carefull here since this would be a breaking change.

**Comment by fanick-ber** [comment]
Sure, but we can restrict giving out the unique url or try again to proxy through the backend. @andre-stefanov can you add the jwt token to a download with the audio player package?

**Comment by andre-stefanov** [comment]
Yes. In Android the exoplayer allows modifying the http headers. iOS has this possibility as well for sure.
